jetty-http2/http2-common/src/main/java/org/eclipse/jetty/http2/HTTP2Cipher.java

```diff
@@ -19,311 +19,310 @@
 package org.eclipse.jetty.http2;
 
 import java.util.Comparator;
-
-import org.eclipse.jetty.util.Index;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class HTTP2Cipher
 {
     public static final Comparator<String> COMPARATOR = new CipherComparator();
 
-    private static final Index<Boolean> __blackProtocols = new Index.Builder<Boolean>()
-        .caseSensitive(false)
-        .with("TLSv1.2", Boolean.TRUE)
-        .with("TLSv1.1", Boolean.TRUE)
-        .with("TLSv1", Boolean.TRUE)
-        .with("SSL", Boolean.TRUE)
-        .with("SSLv2", Boolean.TRUE)
-        .with("SSLv3", Boolean.TRUE)
-        .build();
+    private static final Set<String> __blackProtocols = Stream.of(
+        "TLSv1.2",
+        "TLSv1.1",
+        "TLSv1",
+        "SSL",
+        "SSLv2",
+        "SSLv3"
+    ).map(String::toUpperCase).collect(Collectors.toSet());
```

Comment:
This `String::toUpperCase` uses system default locale.
This will produce undesired side effects on on some, non-US, systems.